### DISCUSSION TITLE
emval::as has unknown type error

### DISCUSSION THREAD

**Discussion by hilarytsai** [mozilla/hubs#2770] [General Questions & Discussion] [answered]
Every time I try to publish my scene, I get an error saying "emval::as has unknown type."  I tried different projects, and even logged into a different Spoke account. It's the same result every time, and I'm unable to publish.  Some of these scenes are very small, so it's not the size.  They were fine to publish before, and I haven't done anything different.  Is this an error on Spoke's side or mine?

**Comment by hilarytsai** [comment, marked as answer]
I would like to say that after restarting Chrome, this issue was gone--not sure what caused it, but I wanted to let you know.

**Reply by robinkwilson** [reply]
Happy to hear! Very interesting error, could be the browser cache or storage had some faulty data. Thanks for telling us your solution!
